textacy/export.py

```diff
@@ -61,14 +65,35 @@ def docs_to_gensim(docs, lemmatize=True,
     """
     gdict = Dictionary()
     gcorpus = []
-    for doc in docs:
-        words = extract.words(doc,
-                              filter_stops=filter_stops,
-                              filter_punct=filter_punct,
-                              filter_nums=filter_nums)
+    stringstore = StringStore()
```

Comment:
What's the purpose of using a StringStore here?  (sorry, not well versed in this.)